### PR TITLE
chore: add Tss Support Admin role

### PR DESCRIPTION
**Pull request by DorisWongNHS**
Draft PR for early review of role name and collect comments if `RolePermission` is needed.

TIS21-3335
TIS21-3339

### CODE REVIEW

File: profile-service/src/main/resources/db/migration/V9.16__add_tss_support_admin_role.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `Role` (`name`)
+VALUES ('TssSupportAdmin')
```
Comment (Judge40) [inline profile-service/src/main/resources/db/migration/V9.16__add_tss_support_admin_role.sql:2]: There are two general conventions for role names, which do we want to follow?
 1. `TssSupportAdmin`
 2. `TSS Support Admin`

**Comment by Judge40** [comment]
You'll need to bump the `profile-service` version too.

File: profile-service/src/main/resources/db/migration/V9.16__add_tss_support_admin_role.sql (hunk 1 above)
Comment (DorisWongNHS) [reply profile-service/src/main/resources/db/migration/V9.16__add_tss_support_admin_role.sql:2]: Yeah, I found there's almost half-half type of conventions for the existing roles in the table. It seems tend to be named without space from the most recent ones. However, if it is not the convention we have to follow, I feel `TSS Support Admin` looks clearer to me. But it is only my thought. What would you prefer?

Comment (Judge40) [reply profile-service/src/main/resources/db/migration/V9.16__add_tss_support_admin_role.sql:2]: While we intend to hide this from UM, if we did want it visible in the future `TSS Support Admin` looks better there.

Comment (DorisWongNHS) [reply profile-service/src/main/resources/db/migration/V9.16__add_tss_support_admin_role.sql:2]: Yep, that makes sense. Will update the role name as `TSS Support Admin`. Thanks @Judge40 .

**Review by Judge40 — APPROVED** [review]

**Pull request merged**